packages/lit-dev-content/site/docs/v2/ssr/client-usage.md

```diff
@@ -48,7 +48,7 @@ For example:
     <!-- App components rendered with declarative shadow DOM placed here. -->
 
     <!-- ssr-client lit-element-hydrate-support should be loaded first. -->
-    <script src="/node_modules/@lit-labs/ssr-client/lit-element-hydrate-support.js"></script>
+    <script type="module" src="/node_modules/@lit-labs/ssr-client/lit-element-hydrate-support.js"></script>
```

Comment:
I was testing on v3, but assume it was the same case for v2 as well?

Reply:
yep! We moved these files here during v2, but it's the same